readme/email_to_note.md

```diff
@@ -0,0 +1,28 @@
+# Email to note
+
+You can save your emails in Joplin Cloud by forwarding your emails to your Inbox Email address. The content of the email and its attachments will be saved in a note with the name of the email inside a notebook called Inbox. 
```

Comment:
Instead of "inbox email" let's use the terms "Joplin Cloud email address"

"The subject of the email will become the note title, and the email body will become the note content".